lib/fs/walkfs.go

```diff
@@ -46,7 +46,7 @@ func (f *walkFilesystem) walk(path string, info FileInfo, walkFn WalkFunc) error
 		return err
 	}
 
-	if !info.IsDir() {
+	if !info.IsDir() && path != "." {
```

Comment:
Why aren't we doing it in the scanner?
This changes semantics of what os.Walk would do, which I am not a fan off.